x-pack/plugins/security_solution/server/endpoint/routes/actions/status.test.ts

```diff
@@ -674,94 +393,21 @@ describe('Endpoint Action Status', () => {
       expect((response.ok.mock.calls[0][0]?.body as any)?.data).toContainEqual({
         agent_id: agentOne,
         pending_actions: {
-          isolate: 0,
+          isolate: getExpected(1),
         },
       });
       expect((response.ok.mock.calls[0][0]?.body as any)?.data).toContainEqual({
         agent_id: agentTwo,
         pending_actions: {
-          isolate: 0,
+          isolate: getExpected(2),
         },
       });
       expect((response.ok.mock.calls[0][0]?.body as any)?.data).toContainEqual({
         agent_id: agentThree,
         pending_actions: {
-          isolate: 0,
+          isolate: getExpected(2), // present in all three actions, but second one has a response, therefore not pending
         },
       });
     });
-
-    describe('with endpoint response index', () => {
```

Comment:
We don't need these tests. we are now only looking at the Endpoint responses index.